### PR TITLE
Exe2: login page, logout button and saving searches

### PR DESCRIPTION
**Pull request by shirrotem**

### CODE REVIEW

File: src/app/component/login/login.component.scss [hunk 1]
```diff
@@ -0,0 +1,21 @@
+
+.login-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    height: 100vh;
+
+    form {
```
Comment (sivanbasa93) [inline src/app/component/login/login.component.scss]: please give on element <form> class name and use in scss 
its more common

File: src/app/component/login/login.component.scss [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      gap: 1rem;
+
+     
+    }
+  
+    h3 {
```
Comment (sivanbasa93) [inline src/app/component/login/login.component.scss]: please give on element h3
class name and use in scss
its more common

File: src/app/component/logout/logout.component.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  templateUrl: './logout.component.html',
+  styleUrls: ['./logout.component.scss']
+})
+export class LogoutComponent {
+
+
+  
+  constructor(private router:Router){}
+
+  logout(): void{
```
Comment (sivanbasa93) [inline src/app/component/logout/logout.component.ts:15]: when we exit from the page , and click logout - removeItem is missing

File: src/styles.scss [hunk 4]
```diff
@@ -6,3 +6,31 @@ body{
     background-color: rgb(239, 216, 202);
 
 }
+
+input {
```
Comment (sivanbasa93) [inline src/styles.scss]: please give on element input
class name and use in scss
its more common

File: src/styles.scss [hunk 5]
```diff
@@ -6,3 +6,31 @@ body{
...
+    border: 1px solid #ccc;
+    border-radius: 5px;
+
+    &:focus {
+      outline: none;
+      border-color: #004b9b; 
+    }
+}
+
+button {
```
Comment (sivanbasa93) [inline src/styles.scss]: please give on element button
class name and use in scss
its more common

Comment (shirrotem) [reply src/styles.scss]: fixed

File: src/app/component/login/login.component.scss (hunk 1 above)
Comment (shirrotem) [reply src/app/component/login/login.component.scss]: fixed

File: src/app/component/login/login.component.scss (hunk 2 above)
Comment (shirrotem) [reply src/app/component/login/login.component.scss]: fixed

File: src/app/component/logout/logout.component.ts (hunk 3 above)
Comment (shirrotem) [reply src/app/component/logout/logout.component.ts:15]: fixed

File: src/styles.scss (hunk 4 above)
Comment (shirrotem) [reply src/styles.scss]: fixed

**Review by sivanbasa93 — APPROVED** [review]

**Pull request merged**